setup.py

```diff
@@ -39,7 +36,9 @@
 
 
 # add platform dependant optional compilation argument
-if sys.platform.startswith('darwin'):
+if sys.platform.startswith('darwin') and not os.environ.get('POT_LEAVE_CC', ''):
+    os.environ["CC"] = "g++"
+    os.environ["CXX"] = "g++"
```

Comment:
why do we need to do this in the first place? POT does not compile with clang or icc?

Reply:
I guess it was intended to fix https://github.com/PythonOT/POT/issues/118 (from #130) for people installing with the system compilers on Mac, but I don't really understand why this would be needed.